src/mame/drivers/equites.cpp

```diff
@@ -379,6 +380,24 @@ D
 #define MSM5232_MAX_CLOCK 6144000
 #define MSM5232_MIN_CLOCK  214000   // unstable
 
+// as described above, MSM5232 clock is from non-crystal circuit (probably 74LS123P),
+// moreover its frequency changes periodically, with the result that the whole 5232
+// audio output is effected vibrato.
+
+// to implement this feature on MAME, I don't know the proper way and where the
+// appropriate loop to control the clock value is, but I found "m_msm->set_clock".
+// applying values from sine-like table at the place, the resulting vibrato frequency
+// depends on the number of elements in the table, and I think its resolution is not
+// enough to secure the accuracy not only in speed but also in pitch.
+
+// so I hope somebody refine this properly.
+
+uint8_t equitesLFOcounter = 0;
+uint8_t equitesLFOdepth;
+uint8_t MSM5232frq;
+
+static const int8_t equitesLFOwaveform[9] = {-128, -98, -43, 42, 97, 127, 77, -13, -83}; // sine like
+
```

Comment:
The actual way the LFO circuit's rate and depth works is with a 32-byte, 8-bit PROM (only the low 5 bits of output are used) whose current address is initially zero, and is incremented on every write to 0xC0FC, see the code at equites.cpp line 428.
The PROM in question is bprom.3h:
	ROM_REGION( 0x0020, "prom", 0 )
	ROM_LOAD( "bprom.3h",  0x00000, 0x020, CRC(33b98466) SHA1(017c73cf8c17dc5047c89316ae5b45f8d22092e8) )
This produces a 32 sample long, 5 bit deep sine wave, which uses an R2R DAC and complicated bit of analog circuitry to vary the clock rate of the msm5232; there is also a trim-pot involved, which probably sets the vibrato 'depth' or perhaps some sort of DC offset, it isn't clear yet.

Reply:
The trimpot (labeled FRQ) actually sets the tuning/center frequency of the LFO for the MSM5232 clock, and annoyingly it isn't clear what it is supposed to be set to.